src/init.cpp

```diff
@@ -1752,6 +1762,8 @@ bool AppInitMain(InitInterfaces& interfaces)
         return false;
     }
 
+    StartupNotify();
+
     // ********************************************************* Step 13: finished
 
     SetRPCWarmupFinished();
```

Comment:
I wonder if you could move this down a few lines so it would only fire after the rpc is up. This would also open the possibility to use it in the test framework to add coverage.

Reply:
Yeah I wasn't entirely sure on the best place to call the function, I could move it to the end of the function.